app/views/items/show.html.haml

```diff
@@ -45,8 +45,7 @@
         = item_field :rights
         -if @item.standardized_rights_statement.present?
           =item_field :standardized_rights_statement
-        = item_field :url, title: 'URL' do
-          = link_to @item.url, @item.url, target: :_blank
+        = item_field :url, title: 'URL'
```

Comment:
Passing the `link_to` block for the `:url` field is no longer necessary b/c the link is now added in the helper method with Rinku.